profile-widget/profilewidget2.cpp

```diff
@@ -1798,49 +1798,21 @@ void ProfileWidget2::repositionDiveHandlers()
 	}
 }
 
-int ProfileWidget2::fixHandlerIndex(DiveHandler *activeHandler)
```

Comment:
I think this is a reasonable change. I always hated this behavior - as I dislike how the 'grow time' behavior "feels"... but that's unrelated.
Please note that the commit message of 4d055d19 is missing a word... "with the mouse, the XXX tried to be smart"